join-chat-to-teams-meeting/Chat Experience/client.js

```diff
@@ -57,51 +43,72 @@ async function init() {
 
   callAgent = await callClient.createCallAgent(tokenCredential);
   callButton.disabled = false;
-
   chatClient = new ChatClient(endpointUrl, new AzureCommunicationTokenCredential(token));
 
   console.log("Azure Communication Chat client created!");
 }
 
 init();
 
+const joinCall = (urlString, callAgent) => {
+  const url = new URL(urlString);
+  console.log(url);
+  if (url.pathname.startsWith("/meet")) {
+    // Short teams URL, so for now call meetingID and pass code API
+    return callAgent.join({
+      meetingId: url.pathname.split("/").pop(),
```

Comment:
This won't be the SDK guidence for how to handle short meeting URLs, once they release their new version next week we should just be using `callAgent.join({ meetingLink: url })` -- should we hold off until this is released? (to avoid updating again?)

Reply:
We could do this -  when will the GA version be out? I would like to have workarounds in case Teams changes break existing code.